Sources/Fuzzilli/Lifting/FuzzILLifter.swift

```diff
@@ -150,6 +150,10 @@ public class FuzzILLifter: Lifter {
             let arguments = instr.inputs.dropFirst().map({ $0.identifier })
             w.emit("\(instr.output) <- CallMethod \(input(0)), '\(op.methodName)', [\(arguments.joined(separator: ", "))]")
 
+        case is CallComputedMethod:
+            let arguments = instr.inputs.dropFirst(2).map({ $0.identifier })
+            w.emit("\(instr.output) <- CallComputedMethod \(input(0)), '\(input(1))', [\(arguments.joined(separator: ", "))]")
```

Comment:
Nit: please remove the `'` around `input(1)` (similar to `LoadComputedProperty`)

Reply:
done